eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -314,6 +314,7 @@ jobs:
 
         if [[ '${{ parameters.buildSourceOnly }}' == 'True' ]]; then
           customBuildArgs="$customBuildArgs --source-only"
+          extraBuildProperties="$extraBuildProperties /p:ReportSbrpUsage=true"
```

Comment:
Looks like this is the default when `--source-only` / `DotNetBuildSourceOnly` is true. Can we set this default in msbuild instead, i.e. in Directory.Build.props? https://github.com/dotnet/sdk/blob/91931824692073c445f634e29f3af8ceab2d04a0/src/SourceBuild/content/Directory.Build.props#L127

Reply:
Or is this too perf sensitive to enable by default? Should we document the switch somewhere?

Reply:
My original mindset was to only run this in CI because that's always where I have always wanted this report.  I don't see value in this for distro maintainers or most dev scenarios.  That being said, it takes less than 4 s in my local test environment.  If we feel there is value to enable this by default, I would treat it more like the prebuilts report and would negate/refactor this check to utilize a SkipSbrpUsageReport property.

```
  <Target Name="ReportSbrpUsage"
          AfterTargets="Build"
          Condition="'$(ReportSbrpUsage)' == 'true'">
```

Reply:
Thanks for elaborating and explaining. I don't feel strongly about this and am fine with either direction.